input/core/src/commonMain/kotlin/majestic/editor/Host.kt

```diff
@@ -0,0 +1,3 @@
+package majestic.editor
+
+data class Host(val tabs: List<ToolBar>)
```

Comment:
I think you confused the comment on variable name, to class name.

The class name should still be called `ToolbarHost`, the variable name can just be `host`